### PR TITLE
[dev-v5] Add .NET 9.0, add version to demo footer

### PR DESCRIPTION
**Pull request by vnbaaij**
- Add .NET 9 (Update TargetFrameworks in `.csproj` files, update `Directory.Packages.props`)
- Fix Home.md typos
- Fix analyzer error in ResourceReader
- Add version info in demo footer (Add `AppVersionService`, update `Directory.Build.props`)
- Update NuGet packages

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -1,46 +1,75 @@
...
-    <AspNetCoreVersion>8.0.7</AspNetCoreVersion>
-    <EfCoreVersion>8.0.7</EfCoreVersion>
+    <RuntimeVersion8>8.0.0</RuntimeVersion8>
+    <AspNetCoreVersion8>8.0.8</AspNetCoreVersion8>
+    <EfCoreVersion8>8.0.8</EfCoreVersion8>
+    <RuntimeVersion9>9.0.0-preview.7.24405.7</RuntimeVersion9>
+    <AspNetCoreVersion9>9.0.0-preview.7.24406.2</AspNetCoreVersion9>
+    <EfCoreVersion9>9.0.0-preview.7.24405.3</EfCoreVersion9>
   </PropertyGroup>
+  <!-- Independent from TargetFrameWorks-->
```
Comment (dvoituron) [inline Directory.Packages.props:11]: Can you add a line of spacing before these “global” comments to have a cleaner file?

File: examples/Demo/FluentUI.Demo.Client/Infrastructure/IAppVersionService.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+namespace FluentUI.Demo.Client;
+
+public interface IAppVersionService
```
Comment (dvoituron) [inline examples/Demo/FluentUI.Demo.Client/Infrastructure/IAppVersionService.cs:7]: I don't think this interface is useful (even if we want to inject it into Services later). The class is directly usable.

File: examples/Demo/FluentUI.Demo.Client/Infrastructure/AppVersionService.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+namespace FluentUI.Demo.Client;
+internal class AppVersionService : IAppVersionService
+{
+    public string Version
+    {
+        get => GetVersionFromAssembly();
+    }
+
+    static public string GetVersionFromAssembly()
```
Comment (dvoituron) [inline examples/Demo/FluentUI.Demo.Client/Infrastructure/AppVersionService.cs:15]: Can you add a comment to explain which version will be used and which string format is returned?

File: examples/Demo/FluentUI.Demo.Client/Layout/DemoMainLayout.razor [hunk 4]
```diff
@@ -42,28 +43,20 @@
...
-<FluentLayoutItem Area="@LayoutArea.Footer" Style="font-size: smaller;">
+<FluentLayoutItem Area="@LayoutArea.Footer" Style="font-size: smaller; display: flex;">
+    Version: @_version
+    &nbsp;-&nbsp;
+    <a href="https://dotnet.microsoft.com/en-us/learn/aspnet/what-is-aspnet-core">Powered by @RuntimeInformation.FrameworkDescription</a>
+    <div style="flex-grow: 1" />
     &copy; Microsoft @DateTime.Now.Year. All rights reserved.
 </FluentLayoutItem>
 
 @code
```
Comment (dvoituron) [inline examples/Demo/FluentUI.Demo.Client/Layout/DemoMainLayout.razor:59]: Can you remove this empty  `@code` ?

**Review by dvoituron — CHANGES_REQUESTED** [review]

File: examples/Demo/FluentUI.Demo.Client/Infrastructure/AppVersionService.cs (hunk 3 above)
Comment (vnbaaij) [reply examples/Demo/FluentUI.Demo.Client/Infrastructure/AppVersionService.cs:15]: Changed this so one would use AppVersion.Version. Added comment there (this is now private)

**Pull request merged**